src/mem/DRAMInterface.py

```diff
@@ -1414,3 +1414,97 @@ class LPDDR5_6400_1x16_8B_BL32(LPDDR5_6400_1x16_BG_BL32):
     tCCD_L = "0ns"
     tRRD_L = "0ns"
     tWTR_L = "0ns"
+
```

Comment:
This should be in a new file (LLMInterface.py)

In fact, there's no need to put this in the `src/` directory either. All of this is configurable at runtime.